README.md

```diff
@@ -15,6 +15,24 @@ Just install gradle and run:
 
     gradle run
 
+### Run with multiple machines
+- Connect machines to a common network
+- Find the ip of the server/host machine (for example `192.168.1.123`)
+- Find a port which can be used for JADE communication (for example `5555`)
+
+For server/host machine
+
+    gradle run --args="-isHost 192.168.1.123 -port 5555 -stage1 -stage2 ... -noTK"
+
+For client machines
+
+    gradle run --args="-host 192.168.1.123 -localPort 5555 -stage3 -noTK"
```

Comment:
I believe the -localPort is to be specified for the host and the -port for the clients.

Reply:
Are you sure? I took a look at my bash history from tuesday evening and found it this way? I also tried running server and client on different terminals with the proposed commands. Could you check with multiple machines and let me know?

Reply:
I noticed that as well. Let me know if you have results from execution.

Reply:
Hi @DharminB , 
We tested the changes with @HBRS-MAAS/jsw and I can confirm that the -localPort is to be specified for the host and the -port for the clients. Please make the change accordingly.